### PR TITLE
fix(adminAPI): localize event type

### PR DESCRIPTION
**Pull request by adlerhurst**
- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [x] All open todos and follow ups are defined in a new ticket and justified
- [x] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [x] Critical parts are tested automatically
- [x] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [x] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by adlerhurst** [comment]
issue for missing translations: https://github.com/zitadel/zitadel/issues/5072

File: internal/static/i18n/en.yaml [hunk 1]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        removed: Secret generator removed
+    sms:
+      configtwilio:
+        activated: Twilio SMS configuration activated
+        added: Twilio SMS configuration added
+        changed: Twilio SMS configuration changed
+        deactivated: Twilio SMS configuration deactivated
+        removed: Twilio SMS configuration removed
+        token:
+          changed: Token of wilio SMS configuration changed
```
Comment (livio-a) [inline internal/static/i18n/en.yaml:1065]:
```suggestion
          changed: Token of Twilio SMS configuration changed
```

File: internal/static/i18n/de.yaml [hunk 2]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        removed: Twilio SMS Konfiguration gelöscht
+        token:
+          changed: Token zu Twilio SMS Konfiguration hinzugefügt
+    smtp:
+      config:
+        added: SMTP Knofiguration hinzugefügt
+        changed: SMTP Knofiguration geändert
+        password:
+          changed: Passwort von SMTP Knofiguration geändert
+        removed: SMTP Knofiguration gelöscht
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1072]:
```suggestion
        added: SMTP Konfiguration hinzugefügt
        changed: SMTP Konfiguration geändert
        password:
          changed: Passwort von SMTP Konfiguration geändert
        removed: SMTP Konfiguration gelöscht
```

File: internal/static/i18n/de.yaml [hunk 3]
```diff
@@ -607,10 +622,16 @@ EventTypes:
...
-        set: Benutzer Metadaten gesetzt
-        removed: Benutzer Metadaten gelöscht
-        removed.all: Alle Benutzer Metadaten gelöscht
+    metadata:
+      set: Benutzer Metadaten gesetzt
+      removed: Benutzer Metadaten gelöscht
+      removed.all: Alle Benutzer Metadaten gelöscht
+    domain:
+      claimed: Domain beansprucht
+      claimed.sent: Domain beanspruchungs Information gesendet
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:631]:
```suggestion
      claimed.sent: Domain Beanspruchungs Information gesendet
```

File: internal/static/i18n/de.yaml [hunk 4]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        changed: E-Mail Vorlage geändert
+      text:
+        added: E-Mail Text hinzugefügt
+        changed: E-Mail Text geändert
+    member:
+      added: Instanzmitglied hinzugefügt
+      changed: Instanzmitglied geändert
+      removed: Instanzmitglied gelöscht
+      cascade:
+        removed: Instanzmitglieder kaskadierend gelöscht
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:982]: we currently always use Member in German translations

File: internal/static/i18n/de.yaml [hunk 5]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        changed: OIDC Einstellung geändert
+    policy:
+      domain:
+        added: Domain Richtlinie hinzugefügt
+        changed: Domain Richtlinie geändert
+      label:
+        activated: Label Richtlinie aktiviert
+        added: Label Richtlinie hinzugefügt
+        assets:
+          removed: Asset von Label Richtlinie gelöscht
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1004]:
```suggestion
          removed: Assets von Label Richtlinie gelöscht
```

File: internal/static/i18n/de.yaml [hunk 6]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        changed: Label Richtlinie geändert
+        font:
+          added: Schriftart zu Label Richtlinie hinzugefügt
+          removed: Schriftart von Label Richtlinie gelöscht
+        icon:
+          added: Symbol zu Label Richtlinie hinzugefügt
+          removed: Symbol von Label Richtlinie gelöscht
+          dark:
+            added: Symbol zu dunkler Label Richtlinie hinzugefügt
+            removed: Symbol von dunkler Label Richtlinie gelöscht
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1014]: we currently don't translate icon and have `(dunkler Modus)`, e.g. `Icon (dunkler Modus) zu Label Richtlinie hinzugefügt`, which I prefer

File: internal/static/i18n/de.yaml [hunk 7]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        logo:
+          added: Logo zu Label Richtlinie hinzugefügt
+          removed: Logo von Label Richtlinie gelöscht
+          dark:
+            added: Logo zu dunkler Label Richtlinie hinzugefügt
+            removed: Logo von dunkler Label Richtlinie gelöscht
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1020]: Modus

File: internal/static/i18n/de.yaml [hunk 8]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        multifactor:
+          added: Multifaktor zu Login Richtlinie hinzugefügt
+          removed: Multifaktor zu Login Richtlinie gelöscht
+        secondfactor:
+          added: Zweitfaktor zu Login Richtlinie hinzugefügt
+          removed: Zweitfaktor von Login Richtlinie gelöscht
+      password:
+        age:
+          added: Password Alter Richtlinie hinzugefügt
+          changed: Password Alter Richtlinie geändert
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1041]:
```suggestion
          added: Passwort Alterungsrichtlinie hinzugefügt
          changed: Passwort Alterungsrichtlinie geändert
```

File: internal/static/i18n/de.yaml [hunk 9]
```diff
@@ -883,12 +936,140 @@ EventTypes:
...
+        complexity:
+          added: Password Komplexitätsrichtlinie hinzugefügt
+          changed: Password Komplexitätsrichtlinie geändert
```
Comment (livio-a) [inline internal/static/i18n/de.yaml:1044]:
```suggestion
          added: Passwort Komplexitätsrichtlinie hinzugefügt
          changed: Passwort Komplexitätsrichtlinie geändert
```

File: internal/api/grpc/admin/event.go [hunk 10]
```diff
@@ -23,26 +17,29 @@ func (s *Server) ListEvents(ctx context.Context, in *admin_pb.ListEventsRequest)
...
 
 func eventRequestToFilter(ctx context.Context, req *admin_pb.ListEventsRequest) (*eventstore.SearchQueryBuilder, error) {
 	eventTypes := make([]eventstore.EventType, len(req.EventTypes))
 	for i, eventType := range req.EventTypes {
 		eventTypes[i] = eventstore.EventType(eventType)
 	}
+	aggregateIDs := make([]string, 0, 1)
+	if req.AggregateId != "" {
+		aggregateIDs = append(aggregateIDs, req.AggregateId)
+	}
```
Comment (livio-a) [inline internal/api/grpc/admin/event.go:42]: don't we need the same check for aggregate_type as well?

**Review by livio-a — APPROVED** [review]

**Pull request merged**